jwst/tests_nightly/general/nircam/test_coron3_1.py

```diff
@@ -0,0 +1,65 @@
+from ci_watson.jwst_helpers import raw_from_asn
+
+from ..resources import NIRCamTest
```

Comment:
I am not in favor of mixing relative and absolute imports. I recommend pick one and stick with it.

Reply:
We do use relative imports everywhere in the `jwst` package.   I would be happy to see use move to absolute imports, at least on the tests.  Shall I file an issue?  😈